backend/editor/entries.py

```diff
@@ -250,7 +250,7 @@ def full_text_search(text):
     """
     # Escape special characters
     normalized_text = re.sub(r"([^A-Za-z0-9 _])", r"\\\1", text)
-    normalized_id_text = re.sub(r"([^A-Za-z0-9 _])", r"\\\1", normalizing(text))
+    normalized_id_text = re.sub(r"(-)", r"\\\1", normalizing(text))

```

Comment:
Do you really think you have to normalize "-" to "\-" for the search to work ?

Reply:
@aadarsh-ram could you look into this before merging (sorry this was a pending comment I forgot about.

Reply:
@alexgarel I'll look into it

Reply:
@aadarsh-ram I will say just add a .replace("-", " ") after normalizing.